src/common/UserAvatar.js

```diff
@@ -46,21 +46,20 @@ function UserAvatar(props: Props) {
 
   return (
     <View>
-      <Touchable onPress={onPress} style={style}>
-        <ImageBackground
-          style={style}
-          source={{
-            uri: avatarUrl.get(PixelRatio.getPixelSizeForLayoutSize(size)).toString(),
-            ...(avatarUrl instanceof FallbackAvatarURL
-              ? { headers: getAuthHeaders(auth) }
-              : undefined),
-          }}
-          resizeMode="cover"
-          /* ImageBackground seems to ignore `style.borderRadius`. */
-          borderRadius={borderRadius}
-        >
+      <Touchable onPress={onPress}>
+        <View>
```

Comment:
Makes sense. At first, I thought we could just remove this `View`, but then I got the `React.Children.only expected to receive a single React element child` from somewhere in `Touchable`.

I think a nice independent cleanup would be to remove the outer, unstyled `<View />` that wraps the `<Touchable />` in this component; that doesn't seem to be doing anything important, and I poked around without it and things looked just the same as before.